### PR TITLE
Panic if permission denied opening skb keyring mobile

### PR DESCRIPTION
**Pull request by patrickxb**

### CODE REVIEW

File: go/libkb/skb_keyring.go [hunk 1]
```diff
@@ -82,8 +82,15 @@ func (k *SKBKeyringFile) loadLocked() (err error) {
 			k.G().Log.Debug("| Keybase secret keyring doesn't exist: %s", k.filename)
 		} else {
 			k.G().Log.Warning("Error opening %s: %s", k.filename, err)
+			if os.IsPermission(err) {
```
Comment (maxtaco) [inline go/libkb/skb_keyring.go]: can you factor this out with the config.json file panic?

Comment (patrickxb) [reply go/libkb/skb_keyring.go]: sure.

**Comment by patrickxb** [comment]
session.json is covered by libkb/json.go.

**Comment by patrickxb** [comment]
For leveldb, there is already some non-trivial error handling:

https://github.com/keybase/client/blob/master/go/libkb/leveldb.go#L163

I'd prefer not to mess with that until we see a similar error.

**Pull request merged**